docker-compose.contributing.yml

```diff
@@ -0,0 +1,72 @@
+version: "3.3"
```

Comment:
Not sure why you added this file and what the name is exactly about? IMO we can just remove this.